scopes/dependencies/dependency-resolver/dependencies/dependency-list.ts

```diff
@@ -50,7 +50,9 @@ export class DependencyList {
   }
 
   findByPkgNameOrCompId(id: string, version?: string): Dependency | undefined {
-    const found = this.dependencies.find((dep) => dep.id === id || dep.getPackageName?.() === id);
+    const found = this.dependencies.find(
+      (dep) => dep.id === id || dep.getPackageName?.() === id || dep.id.startsWith(`${id}@`)
```

Comment:
why was this change needed?

Reply:
@zkochan , because for some reason, components (not packages) are saved with versions as part of their ids.

Reply:
so let's say the dep.id is `button@0.1.0` and I'm searching for dependency "button" (without a version) I won't be able to find it otherwise